### PR TITLE
docs: Add files of Roles  

### LINKED ISSUE #33: docs: Update README.md #28
There are certain changes required in the issue docs: Update README.md #28

1. Copy each teams roles from the README.md
2. Make a file for each role in the respective folders with `role.md`. 
eg. for fabrication engineer cut the complete content in README.md and paste it in a new file `fabrication.md` in `mechanical` folder
3. Provide the links to the teams (Mechanical, computer...) respective README.md in the main README.md only.

README.md must be short with proper directions.

### PR DESCRIPTION
**Pull request by yash1412**
fixes #33

### CODE REVIEW

**Comment by pnijhara** [comment]
@yash1412 you weren't assigned this issue!

File: docs/electronics/BatteryManagement.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Battery Management Engineer
+
+The following document has been modified on 18th October 2018
+[Doc given by Abbas bhai](https://github.com/auvzhcet/Documentation/blob/master/BMS%20for%20lithium%20polymer%20battery%20pack.docx)
```
Comment (syedjawadakhtar) [inline docs/electronics/BatteryManagement.md]: This has already been written in BMS_for_lithium_polymer_battery_pack.md.
Delete this file.

File: docs/electronics/BMS_for_lithium_polymer_battery_pack.md [hunk 2]
```diff
@@ -3,33 +3,40 @@
 Battery monitoring is vital for electric vehicles, because the safety, operation, and the life of the battery depends on the battery system. This attribute is exactly the major function of the battery-management system (BMS)—to check and control the status of battery within their specified safe operating conditions. In this article, a BMS method is has been proposed using various circuits.
```
Comment (syedjawadakhtar) [inline docs/electronics/BMS_for_lithium_polymer_battery_pack.md:3]: Rename filname to bms_Engineer.md

File: docs/electronics/PcbDesigner.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# PCB designer
```
Comment (syedjawadakhtar) [inline docs/electronics/PcbDesigner.md]: Rename file to pcb_Engineer.md
Please use camelCases while naming file

File: docs/marketing/CreativeCoordinator.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Creative Coordinator
```
Comment (syedjawadakhtar) [inline docs/marketing/CreativeCoordinator.md]: Rename file

File: docs/marketing/EventManager.md [hunk 5]
```diff
@@ -0,0 +1 @@
+# Event Manager
```
Comment (syedjawadakhtar) [inline docs/marketing/EventManager.md]: Rename file

File: docs/marketing/Photographer.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Video/Photographer
```
Comment (syedjawadakhtar) [inline docs/marketing/Photographer.md:1]: rename file

File: docs/marketing/SocialMediaManager.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+# Social Media Coordinator
```
Comment (syedjawadakhtar) [inline docs/marketing/SocialMediaManager.md]: rename file

File: docs/marketing/WorkshopCoordinator.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+# Workshop Coordinator
```
Comment (syedjawadakhtar) [inline docs/marketing/WorkshopCoordinator.md]: Rename file

**Comment by yash1412** [comment]
@syedjawadakhtar told me to assign this issue to yourself.

File: docs/marketing/SocialMediaManager.md (hunk 7 above)
Comment (syedjawadakhtar) [reply docs/marketing/SocialMediaManager.md]: Was there a reason to delete Social_Media_Manager.md
If the content is same do not delete the file!

File: docs/mechanical/Fabrication.md [hunk 9]
```diff
@@ -0,0 +1,3 @@
+# Fabrication Engineer
```
Comment (syedjawadakhtar) [inline docs/mechanical/Fabrication.md]: Rename file.
File name must depict its contents.
fabrication_Engineer.md for describing fabrication engineers' role.

**Comment by pnijhara** [comment]
OK! Then resolve conflicts!

File: docs/mechanical/Prop.md [hunk 10]
```diff
@@ -0,0 +1,3 @@
+# Prop Engineer
```
Comment (syedjawadakhtar) [inline docs/mechanical/Prop.md]: Rename File

File: docs/mechanical/Simulation.md [hunk 11]
```diff
@@ -0,0 +1,4 @@
+# Simulation Engineer
```
Comment (syedjawadakhtar) [inline docs/mechanical/Simulation.md]: Rename file

File: docs/mechanical/Torpedo.md [hunk 12]
```diff
@@ -0,0 +1,3 @@
+# Torpedo Engineer
```
Comment (syedjawadakhtar) [inline docs/mechanical/Torpedo.md]: Rename File

**Review by syedjawadakhtar — CHANGES_REQUESTED** [review]
1. Follow proper name formatting techniques
2. Do not delete similar content files

File: docs/marketing/social_Media_Manager.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+
+## **Do's**
+
+- Update any news regarding the club - approving it with the faculty advisor and/or Coordinator.
+- Snippets of Newspaper/reshare others post in which the club is mentioned.
+- News about any team member of the club, when achieving a milestone.
+- Comment/Share other teams of the college.
+- Give constructive response to competitors.
+- Promote the post when needed (eg. When crowdfunding is done)
+- Trends of social media must be opted only when its positively beneficient.
```
Comment (pnijhara) [inline docs/marketing/social_Media_Manager.md:17]: Spelling mistake!

File: docs/marketing/social_Media_Manager.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+1. How much impact have he/she done when the accounts were under her/his supervision
+2. Different audience attracting techniques
+3. Posting and boosting of posts on different handles
+
+These are some of the Do's and Dont's of the social media manager of the Club
+
+## **Do's**
+
+- Update any news regarding the club - approving it with the faculty advisor and/or Coordinator.
+- Snippets of Newspaper/reshare others post in which the club is mentioned.
```
Comment (pnijhara) [inline docs/marketing/social_Media_Manager.md:12]: reshare is not a word!
Look for something else.

**Review by pnijhara — CHANGES_REQUESTED** [review]

**Pull request merged**